README.md

```diff
@@ -2,8 +2,13 @@
 
 # _FastTrackML_
 
-An experiment tracking server focused on speed and scalability, fully compatible
-with MLFlow.
+FastTrackML is an API for logging parameters and metrics when running
+machine learning code, and it is a UI for visualizing the result. The
+API is a drop-in replacement for Mlflow's tracking server, and it
```

Comment:
"MLFlow" is the right spelling I believe....

Reply:
The use "MLflow" on their site